apps/surfer/src/locations/Sidebar.tsx

```diff
@@ -1,21 +1,29 @@
 import { EntrySys, SidebarAppSDK } from '@contentful/app-sdk';
+import { Note } from '@contentful/f36-components';
 import { useSDK } from '@contentful/react-apps-toolkit';
-import { useEffect, useRef } from 'react';
+import { useEffect, useRef, useState } from 'react';
 import { SurferContainer } from '../components/SurferContainer';
-import { useFieldSelection } from '../hooks/useFieldSelection';
+import { useConfigurationDialog } from '../hooks/useConfigurationDialog';
+import { useContentHtml } from '../hooks/useContentHtml';
 import { useSurfer } from '../hooks/useSurfer';
+import { isRichText } from '../hooks/useSurferCompatibility';
 import { SurferContext, SurferRpcCommands, SurferRpcMessage } from '../types';
-import { useContentHtml } from '../hooks/useContentHtml';
-import { useConfigurationDialog } from '../hooks/useConfigurationDialog';
-import { Flex, Note } from '@contentful/f36-components';
+
+const FULL_SIZE_PX = 550;
```

Comment:
NIT:
```suggestion
const FULL_HEIGHT_PX = 550;
```

In other places too